edx_solutions_api_integration/users/views.py

```diff
@@ -938,10 +938,11 @@ def post(self, request, user_id, course_id):
         response_data['user_id'] = user.id
         response_data['course_id'] = course_id
 
-        if request.data['positions']:
+        positions = request.data.get('positions')
```

Comment:
We should validate the  presence of `positions` in the post data. If its not there we should raise 400 http status code.